.gitignore

```diff
@@ -14,3 +14,4 @@ coverage
 test/typescript/.idea/*
 test/typescript/*.js
 test/typescript/*.map
+package-lock.json
```

Comment:
@imZack definitely. Would you like to send a PR? I've it disabled globally, so it's not a big deal for me, but I can see why it's needed for others.